src/NUnitTestAdapterTests/IssueNo24Tests.cs

```diff
@@ -10,6 +10,7 @@ namespace NUnit.VisualStudio.TestAdapter.Tests
 {
     [Explicit]
     [TestFixture]
+    [Ignore("Temporary disabled...")]
```

Comment:
Why is Ignore added, it is already marked Explicit.  We don't intent to enable this test.

Reply:
Sorry, it was a mistake.
I'm going to push again right now.